lib/dockerregistry/storage_driver.go

```diff
@@ -235,7 +265,7 @@ func (d *KrakenStorageDriver) Writer(ctx context.Context, path string, append bo
 }
 
 // Stat returns fileinfo of path
-func (d *KrakenStorageDriver) Stat(ctx context.Context, path string) (driver.FileInfo, error) {
+func (d *KrakenStorageDriver) Stat(ctx context.Context, path string) (info driver.FileInfo, err error) {
```

Comment:
Can we avoid named return values? I think just a
```
var info driver.FileInfo
```
at the top of this function will suffice.